cpp/server/ct-server.cc

```diff
@@ -92,19 +98,39 @@ static const bool cert_dummy =
 
 }  // namespace
 
+void handler(int sig) {
```

Comment:
`util.cc::InitCT()` calls glog's `InstallFailureSignalHandler()` which I think accomplishes the same thing (although covering more signals), is that not working for you?